telco-core/configuration/Makefile

```diff
@@ -0,0 +1,18 @@
+.PHONY: check
+check: compare_crs missing_crs
+
+
+.PHONY: compare_crs
+compare_crs:
+	@CLUSTER_COMPARE=$$(command -v kubectl-cluster_compare); \
+	if [[ -z $${CLUSTER_COMPARE} ]]; then \
+		echo "kubectl-cluster_compare tool isn't installed; please download it from https://github.com/openshift/kube-compare"; \
+		exit 1; \
+	fi; \
+	$${CLUSTER_COMPARE} -r ./reference-crs-kube-compare/metadata.yaml -f ./reference-crs -R || true
+
+
+.PHONY: missing_crs
+missing_crs:
+	./compare.sh reference-crs reference-crs-kube-compare compare_ignore || true
```

Comment:
compare.sh only takes in two arguments. The "compare_ignore" is hard coded.

Reply:
I've got a fix for this in a later commit.

Reply:
Ok. Will lgtm as an "informing" test lane for now.